File: private/conv2fft.m
function c=conv2fft(a,b)
afft=fft2(a);
bfft=fft2(b,size(a,1),size(a,2));
cfft=afft.*bfft;
c=real(ifft2(cfft));
for k=1:2 %dims
    c=ifftshift(c,k);
end

File: python/psfs/library/conv2fft.py
import numpy as np

def conv2fft(a, b):
    afft = np.fft.fft2(a, axes=(0,1))
    bfft = np.fft.fft2(b, s=(a.shape[0], a.shape[1]))
    cfft = afft*bfft[...,None]
    c = np.real(np.fft.ifft2(cfft, axes=(0,1)))
    for k in [0,1]:
        c = np.fft.ifftshift(c, k)

    return c
